tools/metams/suite_metaMS/repository_dependencies.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
```

Comment:
This way of defining a suite is outdated. Use `suite` and `auto_tool_repositories` in shed.yaml like here https://github.com/galaxyproject/tools-iuc/blob/1019bf0fda897582e2bbdc773aebb3e08e285aae/tools/ampvis2/.shed.yml#L14

If there are different owners this might be a solution: https://github.com/galaxyproject/tools-iuc/blob/main/tool_collections/samtools/.shed.yml